package.json

```diff
@@ -13,7 +13,9 @@
     "@oclif/plugin-autocomplete": "^0.3.0",
     "@oclif/plugin-help": "^3",
     "@oclif/plugin-not-found": "^1.2.4",
+    "chalk": "^4.1.1",
     "cli-ux": "^5.6.2",
+    "compare-lists": "^1.0.10",
```

Comment:
Can we remove this? It doesn't seem like it's being used here

Reply:
Removed